config/dictionaries/resource.json

```diff
@@ -36,8 +36,8 @@
         "summary": "Get all applications associated with authenticated user",
         "description": "",
         "method": "GET",
-        "endpoint": "/v6/application?$filter=is_directly_accessible_by__user/any(dau:1%20eq%201)",
-        "filters": ""
+        "endpoint": "/v6/application",
+        "filters": "?\\$filter=is_directly_accessible_by__user/any(dau:1%20eq%201)"
```

Comment:
Thanks for fixing this @vipulgupta2048. 👍 
>  I have missed out on some backlashes here

Btw, if one had the time to do these improvements... (But I don't think it's a priority.) Instead of escaped strings, the `filters` and `data` properties _everywhere_ could be given as JSON objects, which could then get automatically translated into appropriately escaped strings depending on targets:
* Target: curl on bash
* Target: curl on PowerShell
* Target: curl on Windows cmd.exe
* Target: web browser URL bar
* Target: JSON
* Target: Javascript

I think that each of the targets above may need different escaping characters (e.g. single backslash, double backslash, caret instead of backslash, single quotes, double quotes, % escaping, etc), at least in some corner cases.

Currently:
```
        "filters": "?\\$filter=device/uuid%20eq%20'<UUID>'",
        "data": "{\n    \"tag_key\": \"<EXISTING-KEY>\",\n    \"value\": \"<NEW-VALUE>\"\n}"
```
What it could be instead:
```
        "filters": ["device/uuid eq '<UUID>'"],
        "data": {
                "tag_key": "<EXISTING-KEY>",
                "value":  "<NEW-VALUE>"
        }
```